centrosome/zernike.py

```diff
@@ -85,21 +113,27 @@ def score_zernike(zf, radii, labels, indexes=None):
         indexes = np.arange(1,np.max(labels)+1,dtype=np.int32)
     else:
         indexes = np.array(indexes, dtype=np.int32)
-    radii = np.array(radii)
+    radii = np.asarray(radii, dtype=float)
+    n = radii.size
     k = zf.shape[2]
-    n = np.product(radii.shape)
     score = np.zeros((n,k))
     if n == 0:
         return score
-    areas = radii**2 * np.pi
+    areas = np.square(radii)
+    areas *= np.pi
```

Comment:
I'm curious about why this pattern is used (also [here](https://github.com/oleksandr-pavlyk/centrosome/blob/e4c81dd44da02be576e52ff7decc7a79ced978d3/centrosome/zernike.py#L61-L62)). Is there some improvement in making two calls over a single line, e.g., `areas = np.square(radii) * np.pi`.